content/en/docs/contribute/localization.md

```diff
@@ -278,7 +278,7 @@ For an example of adding a new localization, see the PR to enable
 
 To guide other localization contributors, add a new
 [`README-**.md`](https://help.github.com/articles/about-readmes/) to the top level of
-[k/website](https://github.com/kubernetes/website/), where `**` is the two-letter language code.
+[kubernetes/website](https://github.com/kubernetes/website/), where `**` is the two-letter language code.
```

Comment:
The `k/xxx` abbr is acceptable. But it's not bad to expand it to `kubernetes/xxx`.

Reply:
I think it will increase the readability

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Readable

Reply:
I don't think it's an improvement. It's just to increase readability and saving user's one click.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Readable